### PR TITLE
Cover/pkg/ipmi codemoving

### PR DESCRIPTION
**Pull request by ChriMarMe**
Move code from ipmi.go to structures.go and constants.go
Remove unused fdSet function

### CODE REVIEW

File: pkg/ipmi/structures.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package ipmi implements functions to communicate with the OpenIPMI driver
```
Comment (hugelgupf) [inline pkg/ipmi/structures.go:5]: The package comment shall only be in one file per package

File: pkg/ipmi/constants.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package ipmi implements functions to communicate with the OpenIPMI driver
```
Comment (hugelgupf) [inline pkg/ipmi/constants.go:5]: Remove package comment. One package comment per package only.

**Review by hugelgupf — CHANGES_REQUESTED** [review]
One quick fix then lgtm

File: pkg/ipmi/structures.go (hunk 1 above)
Comment (ChriMarMe) [reply pkg/ipmi/structures.go:5]: Whoopsi.

File: pkg/ipmi/constants.go (hunk 2 above)
Comment (ChriMarMe) [reply pkg/ipmi/constants.go:5]: yep

**Review by walterchris — APPROVED** [review]

**Review by hugelgupf — APPROVED** [review]

**Pull request merged**